var/spack/repos/builtin/packages/py-ipython/package.py

```diff
@@ -32,9 +30,6 @@ class PyIpython(PythonPackage):
     depends_on('py-decorator',                  type=('build', 'run'))
     depends_on('py-pexpect',                    type=('build', 'run'))
     depends_on('py-backcall',                   type=('build', 'run'), when="^python@3.3:")
-
-    depends_on('py-appnope', type=('build', 'run'),
-                    when=sys.platform == 'darwin' and
-                            int(platform.mac_ver()[0].split('.')[1]) >= 9)
```

Comment:
I don't think this works, right? You can't use a `when=` that evaluates to True or False, it has to be a string, right?

Reply:
Yes - it must be a string (or spec)

Reply:
CORRECTION: I'm wrong, it can be a boolean. We have unit tests for this.